MPMap.html

```diff
@@ -0,0 +1,241 @@
+<!DOCTYPE html>
```

Comment:
btw, how did you get this working? I get a 403 when I add my mountain project profile: https://www.mountainproject.com/user/201321974/francis-bailey/ in the form